Formula/miniserve.rb

```diff
@@ -16,6 +17,14 @@ class Miniserve < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+
+    system "#{bin}/miniserve --print-completions bash > miniserve.bash"
+    system "#{bin}/miniserve --print-completions fish > miniserve.fish"
+    system "#{bin}/miniserve --print-completions zsh > _miniserve"
+
+    bash_completion.install "miniserve.bash"
+    fish_completion.install "miniserve.fish"
+    zsh_completion.install "_miniserve"
```

Comment:
Can we do this more like how `deno` (for example) does it?

Reply:
OK.